tests-e2e/cypress/support/ui_commands.ts

```diff
@@ -0,0 +1,111 @@
+}
+Cypress.Commands.add('getLastPostId', getLastPostId);
+
+function postInCurrentChannel(message: string): void {
+    // # Type the message in the textbox and type Enter.
+    cy.findByTestId('post_textbox').clear().type(message).type('{enter}');
+
+    // * Verify that the textbox is empty after sending the message.
+    cy.findByTestId('post_textbox').should('have.text', '');
+}
+Cypress.Commands.add('postInCurrentChannel', postInCurrentChannel);
+
+function exportSlashCommand() : void {
+    // # Post /export (with Esc to make sure that the autocompletion is gone)
+    // in the channel.
+    cy.postInCurrentChannel('/export {esc}');
+}
+Cypress.Commands.add('exportSlashCommand', exportSlashCommand);
+
+function archiveCurrentChannel() : void {
+    // # Click on the header dropdown arrow.
+    cy.get('#channelHeaderDropdownIcon').click();
+
+    // # Select the Archive Channel item in the menu.
+    cy.get('#channelArchiveChannel').click();
+
+    // # Confirm the archival of the channel clicking on the Archive button.
+    cy.get('#deleteChannelModalDeleteButton').click();
+}
+Cypress.Commands.add('archiveCurrentChannel', archiveCurrentChannel);
+
+function unarchiveCurrentChannel() : void {
+    // # Click on the header dropdown arrow.
+    cy.get('#channelHeaderDropdownIcon').click();
+
+    // # Select the Unarchive Channel item in the menu.
+    cy.get('#channelUnarchiveChannel').click();
+
+    // # Confirm the unarchival of the channel clicking on the Unarchive button.
+    cy.get('#unarchiveChannelModalDeleteButton').click();
+}
+Cypress.Commands.add('unarchiveCurrentChannel', unarchiveCurrentChannel);
+
+function leaveCurrentChannel() : void {
+    // # Click on the header dropdown arrow.
+    cy.get('#channelHeaderDropdownIcon').click();
+
+    // # Select the Leave Channel item in the menu.
+    cy.get('#channelLeaveChannel').click();
+}
+Cypress.Commands.add('leaveCurrentChannel', leaveCurrentChannel);
+
+function inviteUser(userName: string): void {
+    // # Click on the header dropdown arrow.
+    cy.get('#channelHeaderDropdownIcon').click();
+
+    // # Select the Add Members item in the menu.
+    cy.get('#channelAddMembers').click();
+
+    // # Type the username in the search box.
+    cy.get('#selectItems').type(userName);
+
+    // # Click on the row containing the user.
+    cy.get('#multiSelectList').within(() => {
+        cy.findByText(`@${userName}`).click({force: true});
+    });
+
+    // # Click the Add button.
+    cy.get('#saveItems').click();
+    cy.get('#addUsersToChannelModal').should('not.exist');
+
+    // * Make sure that the Add Members modal is gone.
+    cy.get('#addUsersToChannelModal').should('not.exist');
+
+    // * Verify that there is a system message informing that the user was added
+    // to the channel.
+    getLastPostId().then((lastPostId: string) => {
+        cy.get(`#post_${lastPostId}`).
+            should('contain.text', `@${userName} added to the channel by you.`);
```

Comment:
This is failing on test "ID 17" due to the teammate name display preference of the current user - if not set to `username` as expected here. See `cy.apiSaveTeammateNameDisplayPreference('username')` in the webapp for reference.
![Screen Shot 2020-08-11 at 11 02 25 PM](https://user-images.githubusercontent.com/5334504/89914417-93385800-dc27-11ea-8485-e5c7356189f1.png)

Reply:
Flakiness. Flakiness everywhere!

It should be fixed by https://github.com/mattermost/mattermost-plugin-channel-export/pull/14/commits/2137d3d9c40b27f6c65a75149343d8291c41445b. Thank you again for the catch!